crates/matrix-sdk/src/event_cache/deduplicator.rs

```diff
@@ -41,8 +41,8 @@ impl fmt::Debug for Deduplicator {
 }
 
 impl Deduplicator {
-    const APPROXIMATED_MAXIMUM_NUMBER_OF_EVENTS: usize = 800_000;
-    const DESIRED_FALSE_POSITIVE_RATE: f64 = 0.001;
+    const APPROXIMATED_MAXIMUM_NUMBER_OF_EVENTS: usize = 1_000;
```

Comment:
How does the memory usage look like if we just increase the false positive rate?

Reply:
1GB for 1000 rooms with 100 events each. Oof.

Reply:
12 MB if i lower `APPROXIMATED_MAXIMUM_NUMBER_OF_EVENTS` to 10K. I suppose we can keep 1,000 until proven otherwise.